include/display.h

```diff
@@ -14,24 +15,30 @@
  */
 
 /**
- * @brief Valid video resolutions
+ * @brief Video resolution structure
  */
-typedef enum
-{
-    /** @brief 320x240 mode */
-    RESOLUTION_320x240,
-    /** @brief 640x480 mode */
-    RESOLUTION_640x480,
-    /** @brief 256x240 mode */
-    RESOLUTION_256x240,
-    /** @brief 512x480 mode */
-    RESOLUTION_512x480,
-    /** @brief 512x240 mode, high-res progressive */
-    RESOLUTION_512x240,
-    /** @brief 640x240 mode, high-res progressive */
-    RESOLUTION_640x240,
+typedef struct {
+    /** @brief Screen width */
+    int32_t width;
+    /** @brief Screen height */
+    int32_t height;
```

Comment:
Maybe I'd also mention min/max values for width and height. Later we can improve this if we find more restrictions on allowed values.